### ISSUE TITLE
Sample app is failing to run. Please help!!!

### ISSUE THREAD

**Issue by fshafique** [xcv58/meteor-auth0#15]
I followed the directions to the letter:
From the project folder: `C:\Development\Meteor Projects\meteor-auth0\examples\sample-app`
I ran: `meteor --settings production.settings.json`

And I got an error that I have no idea how to fix:
```
[[[[[ C:\Development\Meteor Projects\meteor-auth0\examples\sample-app ]]]]]

=> Started proxy.
C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\meteor-promise\promise_server.js:218
      throw error;
      ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at relative (path.js:1262:5)
    at ImportScanner._getRelativeImportId (C:\tools\isobuild\import-scanner.js:662:24)
    at each (C:\tools\isobuild\import-scanner.js:632:33)
    at _.each._.forEach (C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._resolve (C:\tools\isobuild\import-scanner.js:621:7)
    at each (C:\tools\isobuild\import-scanner.js:728:29)
    at _.each._.forEach (C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:719:5)
    at each (C:\tools\isobuild\import-scanner.js:812:12)
    at _.each._.forEach (C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:719:5)
    at each (C:\tools\isobuild\import-scanner.js:812:12)
    at _.each._.forEach (C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:719:5)
    at each (C:\tools\isobuild\import-scanner.js:812:12)
    at _.each._.forEach (C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:719:5)
    at each (C:\tools\isobuild\import-scanner.js:812:12)
    at _.each._.forEach (C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:719:5)
    at each (C:\tools\isobuild\import-scanner.js:812:12)
    at _.each._.forEach (C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:719:5)
    at outputFiles.forEach.file (C:\tools\isobuild\import-scanner.js:414:14)
    at Array.forEach (<anonymous>)
    at ImportScanner.scanImports (C:\tools\isobuild\import-scanner.js:412:22)
    at sourceBatches.forEach.batch (C:\tools\isobuild\compiler-plugin.js:1045:17)
    at Array.forEach (<anonymous>)
    at Function.computeJsOutputFilesMap (C:\tools\isobuild\compiler-plugin.js:1013:19)
    at ClientTarget._emitResources (C:\tools\isobuild\bundler.js:1057:8)
    at buildmessage.enterJob (C:\tools\isobuild\bundler.js:828:12)
    at C:\tools\utils\buildmessage.js:359:18
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at ClientTarget.make (C:\tools\isobuild\bundler.js:819:18)
    at C:\tools\isobuild\bundler.js:2929:14
    at C:\tools\isobuild\bundler.js:3018:20
    at Array.forEach (<anonymous>)
    at Function._.each._.forEach (C:\Users\fshafique\AppData\Local\.meteor\packages\meteor-tool\1.6.1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:79:11)
    at C:\tools\isobuild\bundler.js:3017:7
    at C:\tools\utils\buildmessage.js:271:13
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:264:29
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:262:18
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:253:23
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.capture (C:\tools\utils\buildmessage.js:252:19)
    at bundle (C:\tools\isobuild\bundler.js:2910:31)
    at files.withCache (C:\tools\isobuild\bundler.js:2857:32)
    at Object.withCache (C:\tools\fs\files.js:1664:12)
    at Object.exports.bundle (C:\tools\isobuild\bundler.js:2857:16)
    at Profile.run (C:\tools\runners\run-app.js:579:36)
    at Function.run (C:\tools\tool-env\profile.js:490:12)
    at bundleApp (C:\tools\runners\run-app.js:578:34)
    at AppRunner._runOnce (C:\tools\runners\run-app.js:622:35)
    at AppRunner._fiber (C:\tools\runners\run-app.js:880:28)
    at C:\tools\runners\run-app.js:408:12

```

## Please help!!!

**Comment by xcv58** [comment]
I think this is related to https://github.com/meteor/meteor/issues/9755

Could you try to upgrade Meteor?

**Comment by xcv58** [comment]
According to https://github.com/meteor/meteor/releases
You may need to `meteor update --release 1.7-beta.25`

I have no Windows machines so it's really hard to reproduce the bug.

**Comment by fshafique** [comment]
I just tried it. It hasn't helped. My update also froze near the end, but let me try again from scratch. Thanks for all the help -- my last option might be to run it a Linux VM. I don't have anything setup for immediate use.

**Comment by xcv58** [comment]
https://github.com/jshimko/meteor-launchpad This is a lighter solution than a Linux VM. And I usually use it to deploy Meteor app.

**Comment by fshafique** [comment]
Works on Linux!

**Comment by fshafique** [comment]
@xcv58 I just want to share my latest findings with you: [https://github.com/meteor/meteor/issues/9841#issuecomment-384536585](https://github.com/meteor/meteor/issues/9841#issuecomment-384536585)

I think it may have something to do with the colon in the package name "xcv58:auth0-lock".  But I'm comparing it with "kadira:flow-router", because I have no issues with that package.

**Comment by xcv58** [comment]
@fshafique thanks for digging into this. I'll fix this if we can confirm it's caused by the `:` in the path.

**Comment by alvin** [comment]
Can confirm that I was getting the same `TypeError: Path must be a string. Received undefined` startup error until installing the package locally and removing the "xcv58:" prefix
